features/details/src/main/java/com/az/detail/viewmodel/DetailsViewModel.kt

```diff
@@ -62,38 +89,113 @@ class DetailsViewModel(
             val response = commentsRepository.getComments(postId, simplePageData.currentPage, size)
             when (response.status) {
                 Status.SUCCESS -> {
-                    _comments.value = response.data!!.commentList
+                    setItemLoadingView(false)
+                    _items.value = items.value?.plus(response.data!!.commentList)
+                        ?: response.data!!.commentList
                     simplePageData = response.data!!.simplePage
                 }
                 Status.ERROR -> Log.d(TAG, response.message!!)
             }
+            setIsLoading(false)
+        }
+    }
+
+    fun createComment() {
+        if (isCommentInvalid()) {
+            return
+        }
+        viewModelScope.launch {
+            val response =
+                createCommentRepository.createComment(
+                    postId,
+                    CreateCommentRequestData(comment.value!!)
+                )
+            when (response.status) {
+                Status.SUCCESS -> {
+                    hideSoftInput()
+                    initSimplePageData()
+                    clearComment()
+                    clearComments()
+                    getItems()
+                }
+                Status.ERROR -> Log.d(TAG, response.message!!)
+            }
+        }
+    }
+
+    val listener = object : PostDetailListener {
+        override fun onClickLikeButton() {
+            likePost()
+        }
+
+        override fun onClickBookmarkButton() {
+            setBookmark()
+        }
+    }
+
+    private fun likePost() {
+        if (details.value?.detailedPost?.pressLike == true) {
+            // "이미 좋아요를 누른 게시물입니다" 토스트 메시지 출력
+            _details.value = PostDetailData(details.value!!.detailedPost)
+            return
+        }
+        viewModelScope.launch {
+            val response = likePostRepository.likePost(postId)
+            when (response.status) {
+                Status.SUCCESS -> _details.value = PostDetailData(response.data!!.detailedPost)
+                Status.ERROR -> Log.d(TAG, response.message!!)
+            }
+        }
+    }
+
+    private fun setBookmark() {
+        // 북마크 추가/취소 되었습니다 토스트 메시지 출력ㅣ
+        if (details.value?.detailedPost?.pressBookMark == true) {
```

Comment:
취향차이긴 한뒈~, boolean 타입의 비교일 경우
== true를 쓰는 것보다
```suggestion
        if (details.value?.detailedPost?.pressBookMark) {
```
형식으로 보여줄 수도 있따 ><

false의 경우
```
!details.value?.detailedPost?.pressLike
```
처럼 보여주는게 더 좋을 수도 있엉  🐰 

근데 이건 취향차이기도 해서 반영하지 않아도 됨 🍰

Reply:
오 이건 제가 놓치고 간 부분이네요~ 저도 원래 스승님 말씀하신 스타일 선호하는데, 깜빡하고 놓치고 갔어요ㅎㅎ 잡아주셔서 감사합니다~~